docs/monoDrive_home/Camera.md

```diff
@@ -74,80 +102,118 @@ Provides a grayscale camera stream.
        "x": 512.0,
        "y": 512.0
      },
-    "max_distance": 50000.0,
     "dynamic_range":  50,
     "fov": 60.0,
     "focal_length": 9.0,
     "fstop": 1.4,
     "min_shutter":  0.000500,
     "max_shutter":  0.001400,
     "sensor_size":  9.07,
-   "channels" : "gray"
-
+    "channels" : "gray",
+    "debug_draw": false,
+    "annotation": false,
+    "include_tags": false,
+    "include_obb": false,
+    "cull_partial_frame": false,
+    "far_plane": 5000.0,
+    "desired_tags": [
+      "dynamic"
+    ]
  }
-
 ]
 ```
+All values are the same as the RGB camera except: 
 
-<p>&nbsp;</p>
+- **channels:** Used to determine the type of image output. For RGB cameras, this should always be `grayscale`.
```

Comment:
this should be "For Grayscale cameras ..."

Reply:
Fixed.